content/class/Animated.md

```diff
@@ -0,0 +1,80 @@
+**Project NewCaw** \
+Author: Frederick Schenk
+
+# Content/Animated
+
+![API: Twitter 1](https://img.shields.io/badge/API-Twitter%201-lightgrey?style=flat-square) ![API: Twitter 2](https://img.shields.io/badge/API-Twitter%202-blue?style=flat-square) ![API: Mastodon](https://img.shields.io/badge/API-Mastodon-purple?style=flat-square)
+
+```c#
+public interface Animated : Media
+```
+
+`Animated` is a subclass of [`Media`](Media.md) which manages a animated picture (gif) from a platform.
+
+> From a technically standpoint `Animated` is just a video without audio, but since it is intended to loop and has different controls it is a separate subclass.
+
+| API    | Twitter 1 | Twitter 2 | Mastodon |
+| ------ | --------- | --------- | -------- |
+| Object | [Media](https://developer.twitter.com/en/docs/twitter-api/v1/data-dictionary/object-model/entities#media) | [Media](https://developer.twitter.com/en/docs/twitter-api/data-dictionary/object-model/media) | [Attachment](https://docs.joinmastodon.org/entities/attachment/) |
+
+## Properties
+
+> When the media was loaded it returns [`media_item_data`](#field_media_item_data), otherwise it returns `null` and initiates [`load_media`](#method_load_media).
+
+> ### Inherited from [`Media`](Media.md)
+> 
+> [**`id`**](Media.md#property_id)
+> 
+> [**`media_url`**](Media.md#property_media_url)
+> 
+> [**`preview_url`**](Media.md#property_preview_url)
+> 
+> [**`preview_image`**](Media.md#property_preview_image)
+> 
+> [**`media_blurhash`**](Media.md#property_media_blurhash)
+> 
+> [**`alt_text`**](Media.md#property_alt_text)
+> 
+> [**`height`**](Media.md#property_height)
+> 
+> [**`width`**](Media.md#property_width)
+> 
+> [**`load_preview_progress`**](Media.md#property_load_preview_progress)
+> 
+> [**`load_media_progress`**](Media.md#property_load_media_progress)
+
+## Creation Methods
+
+### `Picture.from_json` {#creation_method_from_json}
+
+![API: Twitter 1](https://img.shields.io/badge/API-Twitter%201-lightgrey?style=flat-square) ![API: Twitter 2](https://img.shields.io/badge/API-Twitter%202-blue?style=flat-square) ![API: Mastodon](https://img.shields.io/badge/API-Mastodon-purple?style=flat-square)
+
+```c#
+public Picture.from_json (Json.Object obj)
+```
+
+Creates a new object out of a given Json object from other points.
+
+## Methods
+
+### `load_media` {#method_load_media}
+
+![API: Internal](https://img.shields.io/badge/API-Internal-green?style=flat-square)\
+
+```c#
+public async Gtk.MediaFile load_media ()
```

Comment:
This looks like it's a new GTK4 thing. I think that's going to HUGELY reduce our supportable distros - https://pkgs.org/search/?q=gtk4

Reply:
Well, given that I am not really fast (I hope I can target October for finishing this), I think it is reasonable to make the jump to the next GTK...